source/features/conversation-activity-filter.tsx

```diff
@@ -152,9 +152,9 @@ async function addWidget(header: string): Promise<void> {
 }
 
 async function init(): Promise<void> {
-	// Reset the dropdown state #4997
+	// Reset dropdowns state #4997
 	currentSetting = 'default';
-	select('.repository-content')!.classList.remove('rgh-conversation-activity-is-filtered');
+	(await elementReady('.repository-content'))!.classList.remove('rgh-conversation-activity-is-filtered');
```

Comment:
Because of the `awaitDomReady: false`, the `select` was failing sometimes.

Alternatively the select could be placed at the end of the init (but the icon might flash in red before being reset). Or we could use a `deinit`.